code/modules/paperwork/envelope.dm

```diff
@@ -51,9 +51,7 @@
 			var/tag = uppertext(O.destinations[O.currTag])
 			to_chat(user, "<span class='notice'>*[tag]*</span>")
 			sortTag = tag
-			playsound(src, 'sound/machines/twobeep.ogg', 100, 1)
-			overlays = 0
-			overlays += image(icon = icon, icon_state = "deliverytag")
```

Comment:
You might not want to remove this, it adds the delivery tag

Reply:
I checked and no such icon_state exists in bureaucracy.dmi. It was probably just planned and never got implemented

Reply:
Unless you mean the overlay variable is what enables the delivery tag functionality in the abstract sense of allowing it to be delivered via disposals?

Reply:
We definitely had one at one time... it was a grey square. Ah, I wonder when it got lost?

We should make a new one

Reply:
Okay, I found the problem here. It was a problem, as is usual, of copy/paste.

The icon you're looking for is still called "deliverytag" and it's inside storage.dmi, so change that to `icon = 'storage.dmi'`. The reason this was caused is because someone pasted this code from the delivery package code and it gets its icon from the object itself, so it's searching in storage.dmi.

This is the icon
![image](https://user-images.githubusercontent.com/9782036/79746670-fba5a700-82cf-11ea-8c3a-5e721a73f755.png)

Reply:
Ah I see, I'll fix and push later today